### DISCUSSION TITLE
Sitemap Problem

### DISCUSSION THREAD

**Discussion by mukundkadu0** [arthelokyo/astrowind#158] [Q&A]
Anyone know how to add a sitemap working mine is not working

**Comment by yuxxeun** [comment]
can i see your `sitemap` file-related configuration?

**Comment by mukundkadu0** [comment]
Which Files should i show like

**Reply by yuxxeun** [reply]
by default, if u following astro documentation, and you using [astro sitemap](https://docs.astro.build/en/guides/integrations-guide/sitemap/), show me yopur `astro.config.mjs` file

**Comment by mukundkadu0** [comment]
import path from 'path';
import { fileURLToPath } from 'url';
import { defineConfig } from 'astro/config';
import tailwind from '@astrojs/tailwind';
import sitemap from '@astrojs/sitemap';
import image from '@astrojs/image';
import mdx from '@astrojs/mdx';
import partytown from '@astrojs/partytown';
import compress from 'astro-compress';
import { readingTimeRemarkPlugin } from './src/utils/frontmatter.mjs';
import { SITE } from './src/config.mjs';
const __dirname = path.dirname(fileURLToPath(import.meta.url));
const whenExternalScripts = (items = []) => SITE.googleAnalyticsId ? Array.isArray(items) ? items.map(item => item()) : [items()] : [];

// https://astro.build/config
export default defineConfig({
  site: "https://astrowind.vercel.app",
  base: SITE.basePathname,
  trailingSlash: SITE.trailingSlash ? 'always' : 'never',
  output: 'static',
  markdown: {
    remarkPlugins: [readingTimeRemarkPlugin]
  },
  integrations: [tailwind({
    config: {
      applyBaseStyles: false
    }
  }), sitemap(), image({
    serviceEntryPoint: '@astrojs/image/sharp'
  }), mdx(), ...whenExternalScripts(() => partytown({
    config: {
      forward: ['dataLayer.push']
    }
  })), compress({
    css: true,
    html: {
      removeAttributeQuotes: false
    },
    img: false,
    js: true,
    svg: false,
    logger: 1
  })],
  vite: {
    resolve: {
      alias: {
        '~': path.resolve(__dirname, './src')
      }
    }
  }
});

**Comment by yuxxeun** [comment]
1. you should put `<link rel="sitemap" href="/sitemap-index.xml">` in your `Layout` file.
2. in `robots.txt` file, put this `Sitemap: https://<YOUR SITE>/sitemap-index.xml`
